.gitignore

```diff
@@ -105,3 +105,5 @@ typings/
 
 .vscode
 .DS_STORE
+
+dist
```

Comment:
It's not enough to add it to the gitignore.
This only prevents adding new files. 
You have to remove the dist folder from the git index too.

Reply:
I try to remove the dist folder, this would be my git lesson for today.
Thanks for the comments.